backend/battle/forms.py

```diff
@@ -59,21 +59,26 @@ def clean(self):
         pokemon_1 = self.cleaned_data.get('pokemon_1')
         pokemon_2 = self.cleaned_data.get('pokemon_2')
         pokemon_3 = self.cleaned_data.get('pokemon_3')
-        obj_battle = cleaned_data['battle']
         obj_trainer = cleaned_data['trainer']
-        position_pkn_1 = cleaned_data['position_pkn_1']
-        position_pkn_2 = cleaned_data['position_pkn_2']
-        position_pkn_3 = cleaned_data['position_pkn_3']
+
+        if 'battle' not in cleaned_data:
```

Comment:
If I'm not mistaken this can be made by adding `required=True` on the field definition

Reply:
Here I verify if the battle is valid. The value of the battle always will be passed, so I verify if exists